virtwho/config.py

```diff
@@ -825,8 +859,7 @@ class VirtConfigSection(ConfigSection):
     def __init__(self, section_name, wrapper):
         super(VirtConfigSection, self).__init__(section_name, wrapper)
         self.add_key('type', validation_method=self._validate_virt_type, default='libvirt')
-        self.add_key('virttype', validation_method=self._validate_virt_type, default='libvirt',
-                     destination='virttype')
+        self.add_key('sm_type', validation_method=self._validate_sm_type, default="sam", restricted=True)
```

Comment:
Restricted keys are meant to be those which need to be validated (and otherwise included in the scheme of things) but are considered internal meaning we should not output validation messages about them.